### ISSUE TITLE
Better form styling

### ISSUE THREAD

**Issue by crispinread** [OpenDrupal/drupal-apprenice-1page#3]
Looking for some style direction on this one I guess...

**Comment by hedleysmith** [comment]
I think the form styling looks fine - it's just the mailchimp sign up form. Let me know how you'd like to get it looking better and I can take a look if you like though

**Comment by crispinread** [comment]
Yep me too tbh I think I added this issue as a reminder for me when I just
plonked the mailchimp code in, let's close the issue and reopen if we ever
need to.
